src/lib/autoinstall/dialogs/storage.rb

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "cwm"
+require "cwm/dialog"
+require "autoinstall/widgets/storage/overview_tree_pager"
+require "autoinstall/storage_controller"
+
+Yast.import "Label"
+Yast.import "Popup"
+
+module Y2Autoinstallation
+  module Dialogs
+    # Dialog to set up the partition plan
+    #
+    # @example Edit a partitioning section
+    #   devicegraph = devicegraph = Y2Storage::StorageManager.instance.probed
+    #   partitioning = Y2Storage::Autoinst::PartitioningSection.new_from_storage(devicegraph)
+    #   result = Y2Autoinstallation::Dialogs::Storage.new.run
```

Comment:
Is this example ok? Shouldn't the constructor of `Dialogs::Storage` receive the variable `partitioning`?

Reply:
Nice catch. I'll fix the example, using the `partitioning` variable when instantiating the dialog and removing the extra `devicegraph =` two lines above.